### DISCUSSION TITLE
Adding checkbox to "trigger a request for Renovate to run again"

### DISCUSSION THREAD

**Discussion by qubitz** [renovatebot/renovate#22523] [Suggest an Idea]
We just switched over to use the Renovate docker image to self host for GitHub and noticed that the Dependency Dashboard was missing the "Check this box to trigger a request for Renovate to run again on this repository" checkbox. Obviously this feature omission makes sense given the self hosted nature. However waiting time could be reduced if this checkbox triggered a callback. This callback could be listened to by the hosting server which would in turn trigger Renovate to run again.

This feature request is half checking that I didn't miss this feature within the documentation and half checking its viability/usefulness. I'd be happy to hear of any other ways people found to increase the reaction time of renovate.

### Our setup
Windows Server 2019 VM running a custom docker image which inherits from renovate/renovate:35.91

Edit: clarified hosting setup

**Comment by rarkins** [comment]
How exactly are you "self hosting for GitHub"?

**Reply by qubitz** [reply]
Sorry for the ambiguity. I clarified my original posting with our setup. Let me know if missed anything else.

**Comment by rarkins** [comment]
Renovate by nature is a CLI tool and not an application/server. For a checkbox to trigger a run as you would like, it would mean that a webhook needs to be sent by GitHub to an application/server, which would then need to parse that webhook, understand it, and then get Renovate to run a job right away. That can't happen if you have a simple container simply running Renovate one after another on discovered repos.

In other words: adding the checkbox would be no problem. The problem is that you have no server to listen for the event, so it would achieve nothing except confusion

**Reply by qubitz** [reply]
Thanks for your time and detailed explanation. Seems I need to brainstorm more.
